query/view/restrict_ids.js

```diff
@@ -0,0 +1,17 @@
+// Adds a doc id filter to a query
+// Used to easily understand why a specific document got the score that it did
+module.exports = function (vs) {
+  const restrictIds = vs.var('restrictIds').get();
+
+  // no valid restrictIds to use, fail now, don't render this view.
+  if (!restrictIds || restrictIds.length < 1) {
+    return null;
+  }
+
+  return {
+    ids: {
+      type: '_doc',
```

Comment:
Surprisingly, this parameter does not cause issues on ES7, despite the [ES7.8 docs](https://www.elastic.co/guide/en/elasticsearch/reference/7.8/query-dsl-ids-query.html) not listing the `type` parameter.

In general, I think most `type` related functionality was deprecated in ES6 and completely removed in ES7, but I guess this one is silently ignored for now.

I just opened a ticket to discuss [dropping support for ES6](https://github.com/pelias/pelias/issues/881). If we do drop ES6 support, we should just remove this line. If not, we should _technically_ use the value of `schema.typeName` from [pelias-config](https://github.com/pelias/config/blob/master/config/defaults.json#L73)

Reply:
oh man this took me so long to figure out you meant elasticsearch and not ecmascript.

will plumb schema.typeName through for now?

Reply:
I take that back, plumbing through schema.typeName will be a bit gross. I'd rather drop this field